traiders/backend/api/serializers/parity.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+from ..models import Parity
+
+
+class ParitySerializer(serializers.HyperlinkedModelSerializer):
+    def validate(self, data):
+        if data.get('base_equipment') and not data.get('target_equipment'):
+            raise serializers.ValidationError('You should either specify Base Equipment or Target Equipment')
+        return data
+
```

Comment:
My concern is, base_equipment and and target_equipment fields are just links to the corresponding equipment objects. So the client needs to do 2 more requests, just to get the symbols of the equipments. Maybe we should return the whole equipment objects here to make their lives easier? We can do it like this:
```python
class ParitySerializer(...)
    base_equipment = EquipmentSerializer(read_only=True)
    target_equipment = EquipmentSerializer(read_only=True)
    ...
```